x-pack/plugins/spaces/server/lib/request_interceptors/on_post_auth_interceptor.test.ts

```diff
@@ -39,69 +38,18 @@ describe.skip('onPostAuthInterceptor', () => {
    * commented out due to hooks being called regardless of skip
    * https://github.com/facebook/jest/issues/8379
 
-  beforeEach(async () => {
+   beforeEach(async () => {
     root = kbnTestServer.createRoot();
   });
 
-  afterEach(async () => await root.shutdown());
+   afterEach(async () => await root.shutdown());
 
-  */
+   */
 
-  function initKbnServer(router: IRouter, basePath: IBasePath, routes: 'legacy' | 'new-platform') {
-    const kbnServer = kbnTestServer.getKbnServer(root);
-
-    if (routes === 'legacy') {
-      kbnServer.server.route([
```

Comment:
These tests are currently skipped, but I still removed all the legacy routes related tests